dashboard/config/environments/production.rb

```diff
@@ -96,4 +96,8 @@
   # Whether or not to skip script preloading. Setting this to true
   # significantly speeds up server startup time
   config.skip_script_preload = false
+
+  # The activities_old table should not be part of the schema
+  # DELETE_ME when the activities old table gets removed
```

Comment:
I'll see how Asher is tracking work for the activities table deprecation and add it there